src/sagas/deposit.js

```diff
@@ -189,13 +218,18 @@ export function* requestADeposit() {
     /** @type {Deposit} */
     let deposit
     try {
-        deposit = yield call([tbtc.Deposit, tbtc.Deposit.withSatoshiLotSize], new BN(1000000))
-    } catch (err) {
-        if (err.message.includes(METAMASK_TX_DENIED_ERROR)) return
+        const lotSizeInBtc = yield select(state => state.deposit.lotSize)
+        const lotSizeInSatoshis =
+            new BN(lotSizeInBtc * BitcoinHelpers.satoshisPerBtc.toNumber())
```

Comment:
This is better done as `(new BN(lotSizeInBtc)).mul(BitcoinHelpers.satoshisPerBtc)`. Dropping to a `Number` risks losing precision.

Reply:
Tried a few versions of this but it doesn't work - `BN` doesn't work with decimals so we need to do the multiplication before creating an `BN` instance.

Reply:
Ah… That makes sense. Sats-per-BTC shouldn't lose us any precision anyway, as it's a constant value, so I think we're good overall.